CMakeLists.txt

```diff
@@ -109,11 +109,36 @@ install(TARGETS minizip EXPORT minizip-exports
   LIBRARY DESTINATION "lib"
   ARCHIVE DESTINATION "lib")
 
+set(INSTALL_CMAKE_DIR "cmake")
```

Comment:
The master branch installs the CMake config into `${CMAKE_INSTALL_FULL_LIBDIR}/cmake/${PROJECT_NAME}`, but the 1.2 branch installs the `minizip-exports.cmake` file into the `cmake` directory.

Reply:
What about changing the names to `minizip-exports-1.2.cmake`? Or something along those lines? Or into a directory called `1.2`?

Reply:
That’s kind of redundant information, because we already have a version config file.

My concern is not to be able to install multiple versions into the same installation directory. My concern is regarding consistency.

For consistency with the master branch we should install into ‘lib/cmake/minizip’, but someone might rely on the old installation directory.

If you think, this is not a problem, then I will change it.

Reply:
Ok I understand.